low-level/sensors/impl/platforms/realstor/realstor_enclosure_sensor.py

```diff
@@ -120,6 +138,8 @@ def run(self):
         try:
             # Timeout counter for controller login failed and ws request failed
             mc_timeout_counter = self.rssencl.mc_timeout_counter
+            # check ws_resonse status code to verify if request was successful.
```

Comment:
> \ # check ws_resonse status code to verify if request was successful

and... used to determine what exactly? update that in comment too

Reply:
please check if the new one sounds right.